src/shared/seccomp-util.c

```diff
@@ -689,13 +725,51 @@ const SyscallFilterSet syscall_filter_sets[_SYSCALL_FILTER_SET_MAX] = {
                 "setuid\0"
                 "setuid32\0"
         },
+        [SYSCALL_FILTER_SET_SIGNAL] = {
+                .name = "@signal",
+                .help = "Process signal operations",
+                .value =
+                "kill\0"
```

Comment:
Hmm, `kill` is the only one that is about sending signals, rather then receiving. The .help string also does not apply to `kill`. So maybe it should be excluded from this list, to make the list "safe".

Reply:
indeed. Dropped it here now. kill is part of `@process` anyway, and i figure we don't need it in both. Will force push a new version shortly.